app/lib/tax_return_status.rb

```diff
@@ -45,4 +45,8 @@ def message_templates
   def self.message_template_for(status, locale = "en")
     message_templates[status.to_sym] ? I18n.t(message_templates[status.to_sym], locale: locale) : ""
   end
+
+  def self.statuses_that_count_towards_capacity
```

Comment:
In a quick skim, my eye goes straight towards the excepted statuses. Maybe we can set up a const called `STATUSES_EXCLUDED_FROM_CAPACITY = [:intake_before_consent, :intake_in_progress, :file_accepted, :file_not_filing].freeze`, then update this method to use that const. Might be a bit more readable for the people who come after us!